data/json/professions.json

```diff
@@ -648,6 +656,46 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "boy_shorts" ]
     }
+  }, 
+    {
+    "type": "profession",
+    "ident": "groundskeeper",
+    "name": "Landscaper",
+    "description": "You used to mow lawns and trim hedges for the wealthy. Contract work was getting scarce even before the zombies came, but now you've got nothing left but your tools and expertise.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "fabrication" }, { "level": 2, "name": "survival" } ],
+    "items": {
+      "both": {
+        "items": [ "tank_top", "jacket_windbreaker", "hat_ball", "gloves_work", "glasses_safety", "socks", "sneakers", "pants_cargo" ],
+        "entries": [ { "item": "g_shovel" }, { "item": "scabbard", "contents-item": "machete" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "bra", "panties" ]
+    }
+    },
+  {
+    "type": "profession",
+    "ident": "homemaker",
+    "name": "Nursing Assistant",
+    "description": "You were providing in-home care for the elderly, even as the whole world fell apart around you. You can only pray that you don't see your former clients among the walking dead...",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "firstaid" }, { "level": 1, "name": "cooking" } ],
+    "items": {
```

Comment:
Add a panties for both male and female.

Reply:
What does this mean? The lines for gendered clothing are following the same outline as nearly every other class.

Reply:
And where is the "gendered clothing"? I don't see it. I see it for the landscaper profession, but not for the nursing assistant.

Reply:
that's strange. In the version of this document I'm looking at, there is underwear for both genders.

Reply:
And that what I see:
![default](https://user-images.githubusercontent.com/11132525/53759961-fa9e9400-3eda-11e9-929d-1a911e5b0a75.png)

Reply:
You need to commit and push your changes, then, because I am also not seeing gendered clothing or underwear.